v03_pipeline/lib/tasks/write_relatedness_check_table_test.py

```diff
@@ -57,7 +57,7 @@ def setUp(self) -> None:
 
         # Force imported callset to be complete
         ht = import_vcf(TEST_VCF, ReferenceGenome.GRCh38)
-        ht = ht.annotate_globals(sample_type=SampleType.WGS.value)
+        ht = ht.annotate_globals(validated_sample_type=SampleType.WGS.value)
```

Comment:
is `validated_sample_type` used anywhere?

Reply:
yes!  we check it in the `complete` method of `ValidateCallset`.

Reply:
The change to this block was necessary because we're "mocking" the upstream task completion.